web/src/components/tasks/BusinessFormation.test.tsx

```diff
@@ -923,49 +1017,11 @@ describe("<BusinessFormation />", () => {
       });
 
       describe("when agent number selected", () => {
-        it("agent number", () => {
+        it("agent number", async () => {
           renderWithData({ agentNumber: "", agentNumberOrManual: "NUMBER" });
+          await submitBusinessTab(false);
           expect(userDataWasNotUpdated()).toEqual(true);
         });
-        describe("when agent number selected", () => {
```

Comment:
removing because these tests are duplicates

Reply:
duplicates of what?

Reply:
To address a complicated conflict when rebasing, I ended up accepting both changes and deleted the tests that were duplicates. I missed a few tests.